components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -414,6 +414,13 @@ def main():
             # Remove some less needed fields to reduce output size.
             del model_status['metadata']['managedFields']
             del model_status['status']['conditions']
+            del model_status['address']['url']
```

Comment:
Some of these information can be important for some users, can we have a if statement to check the dictionary size before deleting these fields?

something like `if sys.getsizeof(model_status) > 3000`

Reply:
Got it! how can I test all the changes before push them?

Reply:
Since we don't have unit tests for this component, you can run it on a sample pipeline or put your changes in a helper function and write a test case to verify the if statement.